otel-extension/src/main/scala/io/scalac/mesmer/otelextension/instrumentations/akka/actor/Instruments.scala

```diff
@@ -0,0 +1,83 @@
+package io.scalac.mesmer.otelextension.instrumentations.akka.actor
+import io.opentelemetry.api.GlobalOpenTelemetry
+import io.opentelemetry.api.metrics.LongCounter
+import io.opentelemetry.api.metrics.LongHistogram
+import io.opentelemetry.api.metrics.MeterProvider
+
+trait Instruments {
```

Comment:
So now, the `trait Instruments`, `object Instruments` and `InstrumentsProvider` is just `object Instruments` with some extra steps. We don't replace the instruments in testing as it turned out to be a no-go. 

Please replace with just an `object Instruments`. Please rename it to `AkkaActorInstruments` for more clarity